code/datums/ai_laws.dm

```diff
@@ -263,6 +263,7 @@
 /datum/ai_laws/proc/pick_weighted_lawset()
 	var/datum/ai_laws/lawtype
 	var/list/law_weights = CONFIG_GET(keyed_list/law_weight)
+	law_weights -= /datum/ai_laws/default/asimov
```

Comment:
First, this won't fix it. This is a list of lawsets ids and not types. so it ought to be "asimov", not  `/datum/ai_laws/default/asimov`.
Second, it should be a blacklist arg containing ai_laws typepaths. Their initial id value will be removed from the law_weights this way:
```
	for(var/lawset in blacklist)
		law_weights -= initial(lawset.id)
```
Third, the arg should be unset by default and set to list("asimov") in this call:
https://github.com/tgstation/tgstation/blob/55b92786dcf0f9c5172efd111a6ecbf83ec0dae2/code/datums/ai_laws.dm#L234

Reply:
> Third, the arg should be unset by default and set to `list(/datum/ai_laws/default/asimov)` in this call (the proc is called three times in the code and one of them is a config setting for random weighted lawsets):
> 

is what I did what you meant by having it unset? I wasn't certain on your meaning by that.

Reply:
Not exactly. An argument is in layman terms what you see between the parenthesis of a proc definition (e.g. `thing/proc/does_something(argument)`) or call (`thing.does_something(arg_value)`). Though it doesn't matter anymore since the way it's done now works too.